client/src/Components/sitterProfile/aboutMeProfile/AboutMeProfile.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Paper, Avatar } from "@material-ui/core";
+import "./AboutMeProfile.css";
+import { makeStyles } from "@material-ui/core/styles";
+import { Alert } from "@material-ui/lab";
+const useStyles = makeStyles({
+  profilePic: {
+    width: "140px",
+    height: "140px",
+    alignContent: "center",
+  },
+});
+
+function AboutMeProfile(props) {
```

Comment:
```suggestion
function AboutMeProfile({profile}) {
```

that way you won't need to initialize another `const` on line 16